files/gen_chkrootkit_expected_output.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+LOG_DIR=/var/log/chkrootkit
+
+eval /usr/sbin/chkrootkit -q > $LOG_DIR/log.today.raw 2>&1
+# the sed expression replaces the messages about /sbin/dhclient3 /usr/sbin/dhcpd3
+# with a message that is the same whatever order eth0 and eth1 were scanned
+sed -r -e 's,eth(0|1)(:[0-9])?: PACKET SNIFFER\((/sbin/dhclient3|/usr/sbin/dhcpd3)\[[0-9]+\]\),eth\[0|1\]: PACKET SNIFFER\([dhclient3|dhcpd3]{PID}\),' \
```

Comment:
It's a but unfortunate we need to copy this from the cron script, but I don't know a better way.  Maybe you can add a comment pointing to the file where this comes from, in case we need to change it in the future?